install/kubernetes/cilium/templates/cilium-envoy/daemonset.yaml

```
@@ -69,6 +69,7 @@ spec:
       securityContext:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- include "envoy.initcontainers" . | nindent 6 }}
```

Is this case sensitive?
```suggestion
      {{- include "envoy.initContainers" . | nindent 6 }}
```